.github/workflows/regression.yml

```diff
@@ -209,7 +209,7 @@ jobs:
         uses: docker://ghcr.io/su2code/su2/test-su2:240320-1536
         with:
           # -t <Tutorials-branch> -c <Testcases-branch>
-          args: -b ${{github.ref}} -t develop -c develop -s ${{matrix.testscript}}
+          args: -b ${{github.ref}} -t develop -c fix_aachen_test_case -s ${{matrix.testscript}}
```

Comment:
Fix this please @alecappiello

Reply:
Yes! Though, before I can do that we need to push new mesh and restart files into TestCases/develop branch. Could you please accept that PR (https://github.com/su2code/TestCases/pull/155)?